cli/api/daemon.go

```diff
@@ -423,6 +424,12 @@ func (d *daemon) startMaster() (err error) {
 		}).WithError(err).Fatal("Unable to register master as host")
 	}
 
+	// Load keys if they exist, else generate them
```

Comment:
a) Include a positive Info-level message after this is done
b) Include the file where you're trying to load/create as a field in the logger
c) Maybe distinguish in the logging between creation and loading? Might help later.

```
keylog := log.WithFields(logrus.Fields{
    "keyfile": masterKeyFile,
})
...
keylog.Info("Loaded master keys from disk")
```